### DISCUSSION TITLE
Read timed out. (read timeout=30)

### DISCUSSION THREAD

**Discussion by benzodi4zepine** [Taxel/PlexTraktSync#1426] [Support] [answered]
Hey can anyone help me with this timeout error? I've changed the config command from 30 timeout to 
plex:
  timeout: 360
And it still gives this error.

2023-03-09 01:22:02,791 WARNING[PlexTraktSync]:HTTPSConnectionPool(host='**IP**.plex.direct', port=42429): Read timed out. (read timeout=30) for plextraktsync.plex.PlexLibraryItem.get_guids(), retrying after 1 seconds (try: 1/5)

**Comment by fatshotty** [comment]
I have the same problem :/
any news?

**Comment by fatshotty** [comment, marked as answer]
@BruhIdklol13 I've changed the TIMEOUT value in plexapi library and it seems to work now
hope this can help

**Reply by benzodi4zepine** [reply]
> @BruhIdklol13 I've changed the TIMEOUT value in plexapi library and it seems to work now hope this can help

Hey how do I edit and change the timeout value? can you send me the lines that has it?

**Reply by fatshotty** [reply]
looking into error log, I got the "plexapi" lines that performs the request.
I accepts the TIMEOUT argument; I changed the value hardcoding "500" (number) , saved and restart the plex-trakt-sync script.
that's it.
(I don't remember the file, sorry)

**Reply by schellbell** [reply]
@BruhIdklol13 did you figure out where to change these values? I'm having this issue now and cannot figure it out for the life of me

**Reply by fatshotty** [reply]
hi, you can see the error log and get the `plexapi/server.py` path.
now u can edit the file, going to line 742:
(you should see something like) `timeout = timeout or TIMEOUT`

I've changed it in:
`timeout = 500 # timeout or TIMEOUT`

restarting the tool, it correctly works for me

**Reply by simonc56** [reply]
You can try ~~https://github.com/Taxel/PlexTraktSync/pull/1466~~ version `0.26.4`
